### ISSUE TITLE
Modify toleration to make sure aws-node daemonset is schedule on all nodes

### ISSUE THREAD

**Issue by liwenwu-amazon** [aws/amazon-vpc-cni-k8s#109]
Need to add following 
```
tolerations:
      - operator: "Exists"
         effect: "NoExecute"
      - operator: "Exists"
         effect: "NoSchedule"
```

**Comment by discordianfish** [comment]
```
      tolerations:
      - operator: Exists
```
Should be all that's needed. That wil tolerate everything.

**Comment by vsiddharth** [comment]
https://github.com/aws/amazon-vpc-cni-k8s/pull/128

**Comment by liwenwu-amazon** [comment]
fixed in #128

**Comment by cjbottaro** [comment]
I am using EKS and noticed that when I add a NoExecute taint to a node, it removes the container running this. What do I do to use this updated version that tolerates everything?
